src/main/java/me/totalfreedom/totalfreedommod/command/Command_wipecoreprotectdata.java

```diff
@@ -0,0 +1,31 @@
+package me.totalfreedom.totalfreedommod.command;
+
+import me.totalfreedom.totalfreedommod.rank.Rank;
+import me.totalfreedom.totalfreedommod.util.FUtil;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+import org.bukkit.scheduler.BukkitRunnable;
+
+@CommandPermissions(level = Rank.TELNET_ADMIN, source = SourceType.ONLY_CONSOLE)

```

Comment:
same as wipeflatlands- that's why I guess, and telnet clans wiping maps will need to do this for flatlands.